scripts/paas_app_wrapper.sh

```diff
@@ -5,15 +5,15 @@ case $NOTIFY_APP_NAME in
     exec scripts/run_app_paas.sh gunicorn -c /home/vcap/app/gunicorn_config.py application
     ;;
   delivery-worker-retry-tasks)
-    exec scripts/run_app_paas.sh celery -A run_celery.notify_celery worker --loglevel=INFO --concurrency=11 \
+    exec scripts/run_app_paas.sh celery -A run_celery.notify_celery worker --loglevel=INFO --concurrency=4 \
```

Comment:
@sakisv this should actually be unnecessary, as per [the docs](https://docs.celeryproject.org/en/stable/reference/cli.html#cmdoption-celery-worker-c). In fact, it would be better _not_ to specify it in case our CPU provision changes in future.

Reply:
Agreed to leave this as-is for now: the important thing is that it's relatively low and it's unlikely the number of cores on any worker instance will _decrease_, which was my main concern here.